src/status_im/contexts/shell/view.cljs

```diff
@@ -15,6 +15,15 @@
     (rf/dispatch [:navigate-back])
     true))
 
+;; A hidden view-id-tracker view, required for e2e testing
+(defn view-id-tracker
+  []
+  (let [view-id (rf/sub [:view-id])]
+    [rn/text
+     {:accessibility-label :view-id-tracker
+      :style               {:position :absolute :width 0 :height 0 :z-index -1 :color :transparent}}
```

Comment:
In theory, position and z-index should be sufficient. Should I remove the unnecessary ones? It's just a precaution to avoid issues like text being partially visible while the screen is loading or other edge cases.

Reply:
{
    opacity: 0,
    position: 'absolute',
    height: 0,
    width: 0,
  }
  
  this should work better

Reply:
thank you, updated.

Reply:
Updated code to use `:color :transparent`. Use of height, width or opacity 0 is making view not accessible to appium. More details: https://github.com/status-im/status-mobile/issues/22448#issuecomment-2788383271